src/scripts/eastern_kingdoms/burning_steppes/blackrock_depths/blackrock_depths.h

```diff
@@ -63,6 +63,7 @@ enum
     EVENT_BAR_PATRONS          = 100,
 
     FACTION_DARK_IRON          = 54,
+    FACTION_DARK_IRON_NEUTRAL  = 674,
```

Comment:
In my sniffs, these mobs use only factions 15 and 54. Perhaps 15 is the neutral faction?

Reply:
Cmangos also used 15. If the sniffs say 15 I can change it to that.

Reply:
Do the sniffs show any neutral mobs before the arena is cleared? Currently the spectators and the senators are neutral from the start. From what I see on classic, all spectators should be hostile until the arena is cleared.

Reply:
Updated to 15

Reply:
Updated to use faction 15

Reply:
@ratkosrb sniff data showed that Shadowforge Senators and Arena Spectators were only encountered with a faction id of neutral inside of the Arena cyclinder. See image and query below. This query was run against the sniffs_combined3 database. 

`
SET @X = 595.78;
SET @Y = -188.65;
SET @Z = -38.63;
SET @RADIUS = 69*69;
SET @HEIGHT = 10;

SELECT *, COUNT(*) AS count FROM (
SELECT
case 
when c.id = 8904 then 'Shadowforge Senator'
when c.id = 8916 then 'Arena Spectator'
END AS NAME,
 c.id, c.faction, 
(POWER(c.position_x - @X, 2) + POWER(c.position_y - @Y, 2)) < @RADIUS AND ( c.position_z >= @Z AND c.position_z <= (@Z + @HEIGHT)) AS inArena FROM creature c 
WHERE c.id IN (8904, 8916)) AS a
GROUP BY a.id, a.faction, a.inArena`

![image](https://user-images.githubusercontent.com/3597188/166174920-8d962521-d313-4760-962e-157f0d3c788f.png)